### DISCUSSION TITLE
怎么系统学习前端啊 vuejs有什么好的项目吗？？

### DISCUSSION THREAD

**Discussion by YouthguyZ** [community/community#19616] [Other Feature Feedback, Questions, & Ideas]
好兄弟们推荐一点学习前端编程的书籍和视频吧

**Comment by melvincwng** [comment]
@YouthguyZ，要学习 vue.js，我建议您查看以下资源：
 - [freeCodeCamp](https://youtu.be/FXpIoQ_rT_c)
 - [Traversy 媒体](https://youtu.be/qZXt1Aom3Cs)
 - [Codecademy 付费版](https://www.codecademy.com/learn/learn-vue-js)

 还有其他平台，比如 Udemy，在 vue.js 上有很多有用的课程。可以在那里学习更高级的 vue.js 技术。

希望这有帮助👍🏻

**Comment by Zi-Tao** [comment]
@YouthguyZ  https://vuejs.org/guide/quick-start.html
